### ISSUE TITLE
Edit Display Element isn't functioning.

### ISSUE THREAD

**Issue by booshka207** [smeighan/xLights#2631]
**Describe the bug**
xlights looks to be functioning normally, however, I can not get the Edit Display Elements to function at all.  

**To Reproduce**
Option 1:  Right click on my display elements and selected 'Edit Display Element'.  The box doesn't appear.
Option 2: Clicked on the Display Elements symbol in the toolbar.  The box doesn't appear.
Option 3: Clicked (in the menu bar) View>Windows>Display Element.  The box doesn't appear.

Option 4:  Turned off the computer and rebooted.
Option 5:  I have tried 10 different sequences, just in case they were influencing the issue.

**Expected behavior**
The box should open - as per what happens at 6m21s in this demonstration:  https://www.youtube.com/watch?v=CEBzTL2WVyU

**Screenshots**
I have saved a screenshot, but I don't think it really shows anything of significance.

**Versions (please complete the following information):**
 - OS: Windows 10
 - xLights version: 2021.15 (though I had 2020.## (I forget which release) but it had the same issue.  So I updated to 2021.15 and it hasn't resolved the issue.

**Additional context**
n/a

**Attachments**
n/a

**Comment by AzGilrock** [comment]
The box might be appearing off-screen.  I'm going from memory so not sure if the menu options are these exact names but the usual fix is View -> Windows -> Reset to Defaults followed by View -> Windows -> Dock All

**Comment by booshka207** [comment]
Hi Team.  Getting back to this one, I've undertaken the suggested action, but this didn't resolve the issue.  The Display Element field isn't opening on a different screen.  It doesn't open at all.

I've tried using the right click / Edit Display Element option on the View window (didn't work) and I've also tried selecting the View / Windows / Display element option too.  I've also View / Window / Reset to Default as well as the Dock All.  I've even gone one-by-one on all the Windows options.  Every one of them works (i.e. adds or removes the window), except for Display Element. 

I'm running the current xLights version (20021.27).  I'm very stumped why this isn't working.  Can someone have another look?
